tests/utils.py

```diff
@@ -67,6 +70,7 @@
 
 # helper classes
 class RaiseFileProblem:
+    """Raise ``FileProblem`` ."""
```

Comment:
I know you didn't add this code, but it's a good opportunity to mention this:

I have recently found a way to write a generic exception raiser class. You can just pass it the exception that you want to raise. Reduces some duplicate code. Check here: https://github.com/zavolanlab/zarp-cli/blob/4b93643b8650656b18b9301e91413469bff23e6f/tests/utils.py#L6-L17

Maybe you want to add that and replace the usage of `RaiseFileProblem`, `RaiseMetadataWarning`, `RaiseOSError` and `RaiseValueError`? :)